### PR TITLE
Add Pulpcore 3.28 Config

### PR DESCRIPTION
**Pull request by Odilhao**

### CODE REVIEW

File: configs/pulpcore/3.28.yaml [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+      - name: flat-swig30-el8
+        mode: bare
+      - name: flat-rust-toolset-el8
+        mode: bare
+      - name: flat-llvm-toolset-el8
+        mode: bare
+      - name: centos8-devel
+        mode: bare
+      - name: centos8-configmanagement-ansible-29
+        mode: bare
```
Comment (ekohl) [inline configs/pulpcore/3.28.yaml:48]: I thought we'd stop using this now that ansible-core is in EL8.

File: configs/pulpcore/3.28.yaml [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+        pulpcore-3.28-el8: 0
+    external_repos:
+      - name: flat-base-el8
+        mode: bare
+      - name: flat-python36-el8
+        mode: bare
+      - name: flat-python39-el8
+        mode: bare
+      - name: flat-python39-devel-el8
+        mode: bare
```
Comment (ekohl) [inline configs/pulpcore/3.28.yaml:36]: Will 3.28 be built with Python 3.9? I thought it would use Python 3.11.

Comment (Odilhao) [reply configs/pulpcore/3.28.yaml:36]: Yes, we will keep on 3.9, no rebuild this time

File: configs/pulpcore/3.28.yaml (hunk 1 above)
Comment (Odilhao) [reply configs/pulpcore/3.28.yaml:48]: That's correct, we should drop this and use ansible-core.

**Review by ekohl — APPROVED** [review]

**Pull request merged**